spectral_cube/analysis_utilities.py

```diff
@@ -376,4 +400,7 @@ def stack_cube(cube, linelist, vmin, vmax, average=np.nanmean, convolve_beam=Non
 
     hdu.data = stacked_cube
 
+    if return_cutouts:
+        return hdu, cutout_cubes
```

Comment:
Can a new `SpectralCube` be returned instead of `hdu` ?

Reply:
yes